dev.php

```diff
@@ -187,6 +187,13 @@ function shortcode_ui_dev_advanced_example() {
 			'multiple' => true,
 		),
 		array(
+			'label'    => esc_html__( 'Select Term', 'shortcode-ui-example' ),
+			'attr'     => 'term',
+			'type'     => 'term_select',
+			'taxonomy' => 'post_tag',
+			'multiple' => true,
+		),
+		array(
 			'label'  => esc_html__( 'Background Color', 'shortcode-ui-example' ),
```

Comment:
I was thinking of doing that! Because I've started to looking for examples and noticed that escaping shouldn't be there. But I think that should be a separate PR! Happy to contribute or PR that if you wanna change those? Let me know.. if now I will at some point :)